src/amber/environment/settings.cr

```diff
@@ -8,6 +8,7 @@ module Amber::Environment
     property database_url : String
     property host : String
     property name : String
+    property http_keep_alive : Bool
```

Comment:
Do we want this as a setting? This looks strange as a setting.

Reply:
If we recognize that `keep-alive` is an header some people would want then it makes as much as really any other setting.

Reply:
What about instead making a PIPE for it then. I don't see this as a setting I would make a pipe and have people add or remove the pipe as they wish, this way people can use set the `keep-alive` in the pipelines they want instead globally.

Reply:
Static, web etc are in different pipelines though so it would have to be enabled in both or it would cause problems. Might be the best way to do it though...

Since crystal for some reason turns on keep alive by default, a handler to disable it makes more sense, except it sort of seems like it should be off by default...